docs/graphcast/radios/poi-radio.md

```diff
@@ -88,33 +140,10 @@ To have full control over the POI Radio code and run it directly on your machine
 cargo run
 ```
 
-## Configuration
-
-### Testnet (Goerli)
-
-| Name                         | Example                                                                   |
-| ---------------------------- | ------------------------------------------------------------------------- |
-| `ETH_NODE`                   | `https://goerli.infura.io/v3/<PROJECT_ID>`                                |
-| `PRIVATE_KEY`                | `0x0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef`      |
-| `GRAPH_NODE_STATUS_ENDPOINT` | `http://localhost:8030/graphql`                                         |
-| `REGISTRY_SUBGRAPH`          | `https://api.thegraph.com/subgraphs/name/hopeyen/gossip-registry-test`    |
-| `NETWORK_SUBGRAPH`           | `https://gateway.testnet.thegraph.com/network`                            |
-| `WAKU_HOST` (Optional)       | Defaults to `127.0.0.1`                                                   |
-| `WAKU_PORT` (Optional)       | Defaults to `8546`                                                        |
-| `SLACK_TOKEN` (Optional)     | `xoxp-0123456789-0123456789-0123456789-0123456789` (defaults to `None`)   |
-| `SLACK_WEBHOOK` (Optional)   | `https://hooks.slack.com/services/<ID>/<ID>/<TOKEN>` (defaults to `None`) |
-| `LOG_LEVEL` (Optional)       | Defaults to `INFO`                                                        |
-
-`SLACK_TOKEN` and `SLACK_WEBHOOK` are used for POI divergence notifications in a Slack channel.
-
-`WAKU_HOST` and `WAKU_PORT` specify where the Graphcast node (included in all Radios) runs. If you want to run multiple Radios, or multiple instances of the same Radio, you should run them on different ports. 
-
-If you want to customize the log level, you can set `LOG_LEVEL` to one of `ERROR`, `WARN`, `DEBUG`, `TRACE`.
-
 ## Workflow
 
-Upon initiation, the Radio fetches active allocations of the Radio operator's corresponding Indexer and establishes Radio topics on each allocations identified by subgraph deployment IPFS hash. Also when started, the Radio immediately starts listening for new blocks on Ethereum. On a given interval, the Radio loops through the list and acquires the normalised POI for each deployment (using the metadata of the block that we're on) and saves those nPOIs in an in-memory map. Below we will refer to these nPOIs as _local_ POIs since they are the ones that we've generated.
+The Radio periodically fetches active allocations of the Radio operator's corresponding Indexer and establishes Radio topics on each allocations identified by subgraph deployment IPFS hash. Also when started, the Radio immediately starts polling Graph Node for new blocks on all relevant networks. On a given interval, the Radio loops through the list and acquires the normalised POI for each deployment (using the metadata of the block that we're on) and saves those nPOIs in an in-memory map. Below we will refer to these nPOIs as _local_ POIs since they are the ones that we've generated.
```

Comment:
I think we could add some more explanations, and I was kind of hoping for more changes here.

If I was reading this section to learn how the radio work, I would wonder something like
- when and how often does the radio pull new active allocations? 
- what networks are considered relevant and how are their chainheads updated? how do they affect the subgraphs indexing on respective networks?
- what are the intervals of acquiring data, when does data get compared and cleared? 
- how does one know if the radio is working properly, to know the radio has gotten to the "Voilà!"? 

Maybe see if the answer of these questions should fit in this section, and think about what else should we include to reduce future questions? 

if the paragraphs gets too verbose, chopping it into smaller paragraphs can make it easier to read

Reply:
great points there, I did my best to expand the info while still keeping it fairly high level (lower level details should be in the docstrings and respective cargo docs page imo). But more than happy to get more feedback since the interval explanations was a bit tricky